src/pages/messages/components/TicketListItem.tsx

```diff
@@ -89,7 +93,10 @@ export const TicketListItem = ({ ticket }: TicketListItemProps) => {
           }}>
           <RegistrationListItemContent registration={registrationCopy} ticketView />
           {ticket.type === 'PublishingRequest' ? (
-            <PublishingRequestMessagesColumn ticket={ticket as ExpandedPublishingTicket} />
+            <PublishingRequestMessagesColumn
+              ticket={ticket as ExpandedPublishingTicket}
+              registrationHasFiles={registrationHasFile}
+            />
```

Comment:
Trenger ikke å regne ut `registrationHasFiles` i denne komponenten når sender med all data som trengs for å finne det ut uansett. Bedre å finne begrense data til der det trengs
```suggestion
            <PublishingRequestMessagesColumn
              ticket={ticket as ExpandedPublishingTicket}
            />
```